Sources/FrontEnd/AST+Extensions.swift

```diff
@@ -28,7 +28,7 @@ extension AST {
             return true
           }
         } catch let error as DiagnosedError {
-          fatalError(error.diagnostics.first!.description)
+          fatalError(String(error.diagnostics.lazy.map { "\($0)" }.joined(separator: "\n")))
```

Comment:
There's a custom string interpolation in Utils/ to do that:
```suggestion
          fatalError("\(list: error.diagnostics, joinedBy: "\n")")
```

Reply:
Sorta the way you abhor property wrappers, I never understood or liked custom interpolations.  But I'll use it if you think it's an improvement.